### PR TITLE
Hw2

### PR DESCRIPTION
**Pull request by Ritik3111**

### CODE REVIEW

**Review by timkpaine — CHANGES_REQUESTED** [review]
commits not squashed, remove `.DS_Store` files

File: projects/python/Fly.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### FLY
```
Comment (timkpaine) [inline projects/python/Fly.md:1]: link to project

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: projects/python/Fly.md (hunk 1 above)
Comment (Ritik3111) [reply projects/python/Fly.md:1]: .DS_Store file removed 
Link updated

**Review by timkpaine — CHANGES_REQUESTED** [review]
there is another .DS_Store, and commits should be squashed into 1

**Comment by Ritik3111** [comment]
Done!

**Pull request merged**